src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,31 @@
+.nav {
+  &__list {
+    display: flex;
+    padding: 0;
+  }
+
+  &__item {
+    position: relative;
+    color: #060b35;
+    list-style: none;
+    line-height: 60px;
+
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  &__link {
+    color: inherit;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: 12px;
+    font-weight: 500;
+
+    transition: color $time-transition;
+
```

Comment:
what do you think about making the clickable link area in full height of header? For me its better